src/LibHdb++Cassandra.cpp

```diff
@@ -48,21 +51,20 @@ namespace HDBPP
 //=============================================================================
 HdbPPCassandra::HdbPPCassandra(vector<string> configuration)
 {
-    // LOG(Debug)<<": VERSION: " << version_string << " file:" << __FILE__rev << endl;
-    map<string, string> libhdb_conf;
+    TRACE_LOGGER;
     string local_dc;
     string config;
 
     // defaults
-    Log::LogLevel() = Error;
+    LoggerClass::Log::LogLevel() = Error;
     cass_log_set_level(CASS_LOG_DISABLED);
 
     // convert the config vector to a map
-    string_vector2map(configuration, "=", &libhdb_conf);
+    auto libhdb_conf = extract_config(configuration, "=");
 
     // ---- logging_enabled optional config parameter ----
```

Comment:
I would rename "logging_enabled" configuration parameter to "logging_level"

Reply:
Yes, better reflects its evolution from a switch to dial style variable.  Fixed.